modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -195,6 +195,10 @@ export class {{classname}} {
         {{/isListContainer}}
 
 {{/formParams}}
+        let isWithCredentials: boolean = false;
```

Comment:
Simplify?

`let isWithCredentials: boolean = (this.configuration.withCredentials === true);`